release_notes.md

```diff
@@ -0,0 +1,51 @@
+# Release Notes
+
+## Enhancements
+
+>**IMPORTANT**: This solution pack now requires FortiSOAR version `7.4.0` and later
+
+- Added the tag `CICD` to enhance searchability for the *Continuous Delivery* solution pack on Content Hub
+
+- Aligned the sequence of the task *Setup the Development Environment* with *Setup the Source Control for Production Environment*
+
+- Removed the dependency on the **Attachments** records to create export templates
+
+    - Added *Content and Settings Export Templates* in export template record
+
+- Added a new team *Content Development* to add users involved with content development in FortiSOAR
+
+### Playbook Enhancements
+
+- Added Pull Request (PR) title as an issue comment in *Create Pull Request* playbook
+
+- Pass input record to child playbook in The *Fetch Latest Changes* playbook
```

Comment:
I guess, this is a internal working, do we really need to mention this point here? @cs-sujeet-ahirrao  please confirm

Reply:
Users must know what is happening even if it behind the scenes. However, since it is not user-facing I have not documented it in the SP doc elsewhere.